spicelib/raw/raw_read.py

```diff
@@ -764,6 +764,16 @@ def _load_step_information(self, filename: Path):
                 raise SpiceReadException("Log file '%s' not found" % logfile)
             except UnicodeError:
                 raise SpiceReadException("Unable to parse log file '%s'" % logfile)
+            except EncodingDetectError:
+                try:
+                    encoding = detect_encoding(logfile, r"([\s\S]*)--- Expanded Netlist ---")
```

Comment:
What about modifying the regular expression on the detect_encoding to cover both normal or expanded netlist ? 
What is your opinion on this ?